src/main/java/org/beta/authentification/keycloak/utils/Features.java

```diff
@@ -4,9 +4,17 @@
  */
 package org.beta.authentification.keycloak.utils;
 
+import org.apache.commons.lang.StringUtils;
+import org.keycloak.models.ClientModel;
+
 public class Features {
-    public static boolean isTchapBotEnabled() {
-        return Boolean.parseBoolean(Environment.getenv(Constants.FEATURE_TCHAP_BOT_OTP));
+    public static boolean isTchapBotEnabled(ClientModel clientModel) {
```

Comment:
I would rather add a other method with additional parameters than modify the existing code